charts/rancher-gke-operator/1.0.200/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.8-patch1
```

Comment:
Does this need to be 2.5.9 for rancher changes ? Will Rancher 2.5.8 handle this version.

Reply:
It might have to be 2.5.9 because of https://github.com/rancher/rancher/pull/33473